ui/packages/catalog-ui-search/src/main/webapp/js/CQLUtils.js

```diff
@@ -9,247 +9,287 @@
  * <http://www.gnu.org/licenses/lgpl.html>.
  *
  **/
-/*global define*/
+/*global require*/
```

Comment:
I think you can just remove this line since nobody is enforcing it and later we can add require as a global.